EIPS/eip-5656.md

```diff
@@ -165,11 +164,15 @@ pre (space at byte 8):
 post:
 
 ```
-0000010203040506 080000000000000000000000000000000000000000000000
+0000010203040506 070000000000000000000000000000000000000000000000
 ```
 
 gas used: 6
 
+### Full test suite
+
+[Ethereum Execution Client Specification Tests](https://github.com/ethereum/execution-spec-tests/tree/c0065176a79f89d93f4c326186fc257ec5b8d5f1/tests/cancun/eip5656_mcopy)
```

Comment:
A full suite of tests can be found in the execution spec tests: https://github.com/ethereum/execution-spec-tests/tree/c0065176a79f89d93f4c326186fc257ec5b8d5f1/tests/cancun/eip5656_mcopy